MLS.Agent.Tests/CommandLine/CommandLineParserTests.cs

```diff
@@ -340,7 +341,14 @@ public async Task Verify_argument_specifies_root_directory()
         {
             var directory = Path.GetDirectoryName(typeof(VerifyCommand).Assembly.Location);
             await _parser.InvokeAsync($"verify {directory}", _console);
-            _verifyOptions.Dir.FullName.Should().Be(directory);
+            RelativePath.NormalizeDirectory(_verifyDirectoryAccessor.GetFullyQualifiedRoot().FullName).Should().Be(RelativePath.NormalizeDirectory(directory));
```

Comment:
Is it necessary to call `NormalizeDirectory` on both sides of the assertion? It makes the test a little harder to read and I'm curious about why they wouldn't match.

Reply:
I have added an extension method to do the same